### DISCUSSION TITLE
How I print affiliation of authors while  using output to PDF in project manuscript Quarto 1.4?

### DISCUSSION THREAD

**Discussion by danieltomasz** [quarto-dev/quarto-cli#6424] [Q&A] [answered]
### Description

I am able to get affiliation of authors in html, but no affiliation in PDF and the doc version doesn't include authors at all :( 
I am using quarto 1.4.288

Are there more option regarding authors field? I weren't able to learn much from the page description and its subpages https://quarto.org/docs/manuscripts/

Manuscript looks really promising!

**Comment by mcanouil** [comment]
The default LaTeX template is a default template.
You can see the options/fields used in the template and partials at <https://quarto.org/docs/journals/templates.html#template-partials>.
For instance, the default template only uses the name of the authors.
It also uses the global `institute` key.

Take a look at Journals extension, see <https://quarto.org/docs/extensions/listing-journals>.
In particular, take a look at the partial of the JSS journal template, see <https://github.com/quarto-journals/jss/tree/main/_extensions/jss/partials>.

To note Quarto Manuscripts type of projects uses the "classic" features but focussed on authoring a manuscript.
For specific format options, you still have the guide and references pages.

**Reply by danieltomasz** [reply]
Maybe I misunderstood something I am trying to play with example generated by vscode (q 1.4.288)
I am inserting

```
format:
  pdf: 
    template-partials:
      - title.tex
 ``` 

into `_quarto.yml`

with  slightly changed  content of  `title.tex`from the examples above
```
\date{$date$}
\title{$title$}
\author{$for(author)$$author$$sep$ \and $endfor$}
\date{$date$}
```
But it doesnt  change the resulting pdf (another date doesn't appear in maketitle)

[TEST-MANUSCRIPT.zip](https://github.com/quarto-dev/quarto-cli/files/12255202/TEST-MANUSCRIPT.zip)

**Reply by mcanouil** [reply]
Please avoid sharing text files as archive.
you can share document as follow which is way easier for everyone to reuse using a simple copy/paste.

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

---

This being said below a working example:

````qmd
---
title: "Quarto Playground"
author:
  - Author1
  - Author2
format:
  pdf:
    template-partials:
      - title.tex
---

This is a playground for Quarto.
````

<table>
<tr><th><code>title.tex</code></th><th>PDF</th></tr>
<tr>
<td>

````latex
\date{$date$}
\title{$title$}
% \author{$for(author)$$author$$sep$ \and $endfor$}
\date{$date$}

````

</td>
<td>
<img width="687" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/47e46aa3-5905-490a-81f1-8b91f1667e6d">

</td>
</tr>
<tr>
<td>

````latex
\date{$date$}
\title{$title$}
\author{$for(author)$$author$$sep$ \and $endfor$}
\date{$date$}

````

</td>
<td>
<img width="687" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/be3b5ac0-4b0a-4723-929a-6586b76c36f8">
</td>
</tr>
</table>

**Reply by danieltomasz** [reply]
Thanks!

**Reply by dragonstyle** [reply]
(The file in your archive has a leading space in its name, FYI - I think that is the problem).

**Reply by fuhrmanator** [reply]
I'm late to this party, but wanted to do something simple in the `title.tex` partial for a customized tech report. It's not obvious how to access the YAML metadata inside the template (I was trying to respect the YAML described at https://quarto.org/docs/journals/authors.html#author-metadata). 

Here's how I (with help from copilot) got it work (using `article` document class), at least for the affiliation name, without using a sub-partial). `it` is the current item in a pandoc template loop.

```
\author{\sf$for(by-author)$\ignorespaces
  \Large\textbf{$by-author.name.literal$}%
  $if(by-author.affiliations)$
    $for(by-author.affiliations)$
      $if(it.name)$\\\large $it.name$$endif$$break$
    $endfor$
  $endif$
  \unskip\\[0.5cm]
$endfor$}
```

Edit: I realize OP asked about manuscript, but the general problem of accessing affiliation is what I'm trying to help with.

**Comment by danieltomasz** [comment]
Thanks @mcanouil and I appreciate the links!
Would be useful for a Manuscript  type to use more advanced template?
It would  be useful to have this unified, html uses "affiliation" 

In 1.3 I was using the extensions to polish more my manuscripts, but for me it make sense to add more features to "Manuscript"  as 1.4 will be closer to be  release (or document its limitations, currently, this question can be moved as FR  on issue tracker

**Reply by mcanouil** [reply]
The thing is for each journal there is a different latex template, thus the extensions.
Manuscript projects are compatible with extensions, _i.e._, use the journal extension (if it exist or create it unfortunately) inside your Manuscript project.

**Reply by dragonstyle** [reply, marked as answer]
I think it would be nice if our default PDF template showed additional metadata, for the sake of clarity, but I agree that practically speaking, users will often just use the journal extensions. 

The goal of the various journal specific extensions is to be able to provide a preview of what the typeset version of the article will look like specifically in the specified journal. I think this will mean that manuscript projects will inherently often be accompanied by a journal specific format extension (or more than one).

If you have extensions or features that you think would make sense in Manuscript projects, I'd love to hear it - it's very early in the development of that project so it is still evolving based upon how people are using it!

**Reply by danieltomasz** [reply]
I understand  the need of extensibility but on the other hand  would be nice to have something that works out of box even before the journal is choosen ( sometimes this is done in the end of the writing pipeline); 

so some kind of displaying author institution (extension of default template to reuse html fields at least) would be  a minimal addition (even if users need to add later journal specific fields)

**Reply by danieltomasz** [reply]
to give more examples and ideas (it is just brainstorming, not expecting to implement them), 
- ability to set some options from papaja might be useful https://frederikaust.com/papaja_man/r-markdown-components.html#yaml-front-matter 
-  better support for doc output might be useful, as people often share with collaborators who doesn't use quarto/rmarkdown/latex 
- more examples of workflow online (incorporating plugins), maybe a tutorial how to work with latex template and where different part of the template to enable it to work with quarto (for example  now I want to por myself this  bioarxiv template to quarto   https://github.com/quantixed/manuscript-templates ) 

2 most important
- Maybe out of the box good looking pdf/doc with affiliation and some light polish
-  Tutorials and more examples how work with  or port existing latex templates and create  a quarto extension  on top of manuscript (it is already there as @mcanouil  posted but I think it spread over different pages)


this could combined - as a part of official docs, installation of really simple extension (like adding authors), and explanations for the users how the could extend it on examples (at least for me it would be useful), some kind of simple tutorial of porting tex templates into manuscript with minimal example

**Reply by danieltomasz** [reply]
Edit: I am going through this tutorial https://quarto.org/docs/journals/formats I can share where I was struggling
